aesd-char-driver/aesd-circular-buffer.c

```diff
@@ -100,3 +100,31 @@ void aesd_circular_buffer_init(struct aesd_circular_buffer *buffer)
 {
     memset(buffer,0,sizeof(struct aesd_circular_buffer));
 }
+
+size_t aesd_circular_buffer_size(struct aesd_circular_buffer *buffer) 
```

Comment:
num_entries might be a better name to more clearly distinguish this function from the next.